FOSS/content/AccessTokenManagement/Advanced/client_assertions.md

```diff
@@ -40,6 +40,11 @@ public class ClientAssertionService : IClientAssertionService
                     { JwtClaimTypes.JwtId, Guid.NewGuid().ToString() },
                     { JwtClaimTypes.Subject, options.ClientId! },
                     { JwtClaimTypes.IssuedAt, DateTime.UtcNow.ToEpochTime() }
+                },
+
+                AdditionalHeaderClaims = new Dictionary<string, object>
+                {
+                    { JwtClaimTypes.TokenType, "client-authentication+jwt" }
```

Comment:
```suggestion
                    { JwtClaimTypes.TokenType, "client-assertion+jwt" }
```

@maartenba will merge if internal chat about this value is confirmed. Adding the suggestion here as a todo.